lti_consumer/tests/unit/test_lti_consumer.py

```diff
@@ -825,6 +827,17 @@ def test_context_keys(self):
         for key in context_keys:
             self.assertIn(key, context)
 
+    @ddt.data('a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol', 'strong', 'ul', 'img')
```

Comment:
We can use `allowed_tags` from the lti_consumer/lti_consumer.py file instead of writing all the tags again. Also, since we made the changes for img tag(and the allowed attributes), so the test should be specific to those changes as well.

Reply:
`allowed_tags` cannot be reuses because it is scoped to `function` only and I have added test for `img` src.